package.json

```diff
@@ -76,7 +76,7 @@
     "@uiw/react-codemirror": "^4.22.2",
     "d3-scale": "^4.0.2",
     "d3-time-format": "^4.1.0",
-    "date-fns": "^2.28.0",
+    "date-fns": "^3.6.0",
```

Comment:
@avatus What's the reason behind closing the updates for date-fns, react-dnd and react-dnd-html5-backend?

Reply:
I am collecting them into a single PR. will link when the update is complete